.idea/misc.xml

```diff
@@ -25,7 +25,7 @@
       </value>
     </option>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_7" default="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```

Comment:
Why did we downgrade JDK?

Reply:
Autogen, I didn't touch that. Studio did it for me.

Reply:
It did it for me locally, I just didn't push it to remote.

Reply:
If studio is doing it, it's doing it for a reason.